underpants.js

```diff
@@ -121,15 +188,32 @@ var _ = {};
 * Objectives:
 *   1) if <collection> is an array, call <function> once for each element
 *      with the arguments:
-*         the element, it's index, <collection>
+*         the element, its index, <collection>
 *   2) if <collection> is an object, call <function> once for each property
 *      with the arguments:
-*         the property's value, it's key, <collection>
+*         the property's value, its key, <collection>
 * Examples:
 *   _.each(["a","b","c"], function(e,i,a){ console.log(e)});
 *      -> should log "a" "b" "c" to the console
 */
 
+// accepts a collection of values and a function
+// if collection is array, call function once per element
+// w/ arguments of element, its index, <collection>
+// if collection is an object, call function once per property
+// w/ arguments of property value, its key, <collection>
+
+_.each = (col, func) => {
+    if (Array.isArray(col)) {
```

Comment:
```suggestion
    if (_.typeof(col) === "array") {
```

Although you cannot use the native typeof, since it gives back "object" for arrays, you could implement the `_.typeof` method that you built out above